src/items/external-blocks.md

```diff
@@ -198,6 +201,19 @@ The default for this modifier is `-whole-archive`.
 More implementation details about this modifier can be found in
 [`whole-archive` documentation for rustc].
 
+#### `dylib` versus `raw-dylib`
+
+On Windows, linking against a dynamic library requires that an import library
+is provided to the linker: this is a special static library that declares all
+of the symbols exported by the dynamic library in such a way that the linker
+knows that they have to be dynamically loaded at runtime.
```

Comment:
raw-dylib can be extended to other platforms in the future. For example on macOS there are .tbd files which are roughly equivalent to import libraries. And on ELF based systems it should be possible to synthesize an .so file with SONAME set to the name of the target dylib and all symbols declared with a fake value. The RFC for raw-dylib mentions other platforms as future possibility. Maybe you could rewrite this section a bit to note that other platforms have similar requirements that raw-dylib could fix, but that this is only implemented on Windows currently?

Reply:
Added a note about other platforms and only Windows being supported at the moment.